devops/local_conf/nginx.conf

```diff
@@ -39,12 +41,13 @@ http {
         location /assets {
             max_ranges 0;
             expires 30d;
-            add_header "Access-Control-Allow-Origin"  *;
+            add_header 'Access-Control-Allow-Origin' '*';
+
             # Preflighted requests
             if ($request_method = OPTIONS ) {
-                add_header "Access-Control-Allow-Origin"  *;
-                add_header "Access-Control-Allow-Methods" "GET, POST, OPTIONS, HEAD, PUT, DELETE";
-                add_header "Access-Control-Allow-Headers" "*";
+                add_header 'Access-Control-Allow-Origin' '*';
+                add_header 'Access-Control-Allow-Methods' 'GET, POST, OPTIONS, HEAD, PUT, DELETE';
+                add_header 'Access-Control-Allow-Headers' '*';
                 add_header 'Access-Control-Max-Age' 1728000;
```

Comment:
Same question as above (line 25): Does this actually override the browser limits enforced in FF and Chromium?

Reply:
fixed in e4bfad207150aaafcb0f875b914b3b5cd885d8f0